### PR TITLE
When adding custom close button. Default one stay usable

### PR DESCRIPTION
**Pull request by crochefort**
When we add a custom close button, it show 2 close buttons

## What does this pull request (PR) do? / Que fait cette demande « pull » (PR)?
Fix only class overlay-close is needed for overlay not lightbox

- [x ] Create/update documentation /// Créer/mettre à jour la documentation
- [x ] Build and test PR's code /// Rouler le script de compilation et tester le code de la PR
- [ ] Validate changes against WCAG for accessibility /// Valider les changements avec WCAG pour l'accessibilité
- [ ] Ensure documentation is bilingual /// S'assurer que la documentation soit bilingue

**Screenshots / Captures d'écrans**
image

@duboisp

### CODE REVIEW

**Review by duboisp — CHANGES_REQUESTED** [review]
@crochefort can you also make the adjustment in the instruction on how to add the close button here: 
* https://github.com/wet-boew/wet-boew/blob/master/src/plugins/overlay/overlay-en.hbs#L226
* https://github.com/wet-boew/wet-boew/blob/master/src/plugins/overlay/overlay-fr.hbs#L228

Thank you

**Comment by crochefort** [comment]
@duboisp No need to edit the documentation since it was OK in the doc.

**Comment by duboisp** [comment]
@crochefort Can you update the following line in the working example page:
For: https://github.com/wet-boew/wet-boew/blob/master/src/plugins/overlay/overlay-en.hbs#L230
Currently
```html
&lt;button class=&quot;btn btn-primary popup-modal-dismiss&quot; type=&quot;button&quot;&gt;Close overlay/modal&lt;/button&gt;
```
Need to be updated like:
```html
&lt;button class=&quot;btn btn-primary overlay-close&quot; type=&quot;button&quot;&gt;Close overlay/modal&lt;/button&gt;
```
Change is to replace "popup-modal-dismiss" with "overlay-close".
Similar change is also required for the French working example

**Comment by crochefort** [comment]
@duboisp  Should be ok I think!

File: src/plugins/overlay/overlay.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ var componentName = "wb-overlay",
 				footer = $elm.find( ".modal-footer" )[ 0 ];
 
 				var hasFooter = ( footer && footer.length !== 0 ) ? true : false,
-					hasButton = hasFooter && $( footer ).find( closeClass ).length !== 0,
+					hasButton = hasFooter && $( footer ).find( “.” + closeClass ).length !== 0,
```
Comment (duboisp) [inline src/plugins/overlay/overlay.js:63]: Can you use the vertical double quote?
```suggestion
					hasButton = hasFooter && $( footer ).find( "." + closeClass ).length !== 0,
```

**Review by duboisp — CHANGES_REQUESTED** [review]

**Review by duboisp — APPROVED** [review]

**Pull request merged**